tools/marine_omics/.shed.yml

```diff
@@ -0,0 +1,15 @@
+categories:
+    - Ecology
+owner: ecology
+remote_repository_url: https://github.com/galaxyecology/tools-ecology/tree/master/tools/marine_omics
+homepage_url: https://github.com/fair-ease/Marine-Omics-Galaxy
+long_description: |
+  Tool for identifying biosynthetic gene clusters (BGCs) in genomic & metagenomic data
+type: unrestricted
+auto_tool_repositories:
+  name_template: "{{ tool_id }}"
+  description_template: "Wrapper for marine omics tool: {{ tool_name }}."
```

Comment:
should that not be sanntis? also above the link to the github?

Reply:
oh yes I went to quickly on that